### PR TITLE
Introduction of Seeq package to Conda-Forge

### PR DESCRIPTION
**Pull request by sethgilchrist**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by sethgilchrist** [comment]
@totallyna @mpderbec
Confirming that you are willing to be maintainers on the Seeq package.

**Comment by mpderbec** [comment]
I confirm that I would like to be a maintainer on the `seeq` package.

**Comment by totallyna** [comment]
I confirm that I would like to be a maintainer on the seeq package.

File: recipes/seeq/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  sha256: ab83336f21191c9c66026315de2b2889f3f86cfc44e1475df7b62895be9c1530
+
+build:
+  number: 0
+  skip: True  # [py<36]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
```
Comment (chrisburr) [inline recipes/seeq/meta.yaml]: If (and only if) this is made `noarch: python`:
```suggestion
    - python >=3.6
```

File: recipes/seeq/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    - pip
+  run:
+    - python
```
Comment (chrisburr) [inline recipes/seeq/meta.yaml]: If (and only if) this is made `noarch: python`:
```suggestion
    - python >=3.6
```

File: recipes/seeq/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ab83336f21191c9c66026315de2b2889f3f86cfc44e1475df7b62895be9c1530
+
+build:
+  number: 0
+  skip: True  # [py<36]
```
Comment (chrisburr) [inline recipes/seeq/meta.yaml]: Looks like this can be noarch:
```suggestion
  noarch: python
```

**Comment by sethgilchrist** [comment]
Hi @chrisburr, Thanks for the improvements to the ymal file. I was a bit confused when the CI bot asked me to remove the `noarch` flag.  I see now what's needed.

As for the license file, I will have to do the rounds at work here to see about updating that.

**Pull request closed without merging**